app/managers.py

```diff
@@ -0,0 +1,49 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self._cursor = self._connection.cursor()
+
+    def create(self, first_name: str, last_name: str) -> Actor:
+        self._connection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        cursor = self._connection.execute(
+            "SELECT * FROM actors"
+        )
+        return [Actor(*row) for row in cursor]
+
+    def update(self, actor_id: int,

```

Comment:
In the `update` method, ensure that at least one of `first_name` or `last_name` is provided. Otherwise, the SQL command will be invalid. Consider adding a check to handle this case.